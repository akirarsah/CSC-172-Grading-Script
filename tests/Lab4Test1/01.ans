Error occurred while inserting
Error occurred while inserting
0       DNA     AATTCCGGAATTCCGG
2       RNA     UAGACAUGGAUU
4       DNA     AATTCCGGAATTCCGG
No sequence to remove at specified position
0       DNA     AATTCCGGAATTCCGG
2       RNA     UAGACAUGGAUU
No sequence to print at specified position
DNA     AATTCCGG
End index is out of bounds
Start index is out of bounds
0       DNA     AATTCCGG
1       DNA     TACAGACAT
2       RNA     UAGACAUGGAUU
No sequence to copy at specified position
0       DNA     AATTCCGG
1       DNA     AATTCCGG
2       RNA     UAGACAUGGAUU
Cannot transcribe RNA
No sequence to transcribe at specified position
RNA	UUAAGGCC
