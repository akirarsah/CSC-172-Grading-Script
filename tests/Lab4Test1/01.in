20 /Lab4Test1.txt