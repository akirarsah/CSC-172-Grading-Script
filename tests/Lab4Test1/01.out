AATTCCGGAATTCCGG DNA 0
UAGACAUGGAUU RNA 2
AATTCCGGAATTCCGG DNA 4
AATTCCGGAATTCCGG DNA 0
UAGACAUGGAUU RNA 2
AATTCCGGAATTCCGG DNA
UAGACAUGGAUU RNA
AATTCCGG DNA
AATTCCGG DNA 0
TACAGACAT DNA 1
UAGACAUGGAUU RNA 2
ERROR: Cannot copy empty or invalid sequence
AATTCCGG DNA 0
AATTCCGG DNA 1
UAGACAUGGAUU RNA 2
ERROR: Cannot transcribe empty or invalid sequence
CCGGAAUU RNA
